core/src/avm2/globals/XML.as

```diff
@@ -1,4 +1,42 @@
 package {
     [Ruffle(InstanceAllocator)]
-    public final dynamic class XML {}
+    public final dynamic class XML {
+        public function XML(value:* = undefined) {
+            this.init(value);
+        }
+
+        private native function init(value:*): void;
+
+        AS3 native function name():Object;
+        AS3 native function localName():Object;
+        AS3 native function toXMLString():String;
+        
+        AS3 native function toString():String;
+    }
+}
+
+XML.prototype.name = function():Object {
```

Comment:
I think the following definitions can be moved inside the `class` body, like this:
```as
public final dynamic class XML {
    ...

    prototype.name = function(): Object {
        ...
    }
}
```

Reply:
Fixed